modules/analysis/src/endpoints/test.rs

```diff
@@ -19,36 +18,29 @@ async fn test_simple_retrieve_analysis_endpoint(
     let uri = "/api/v2/analysis/root-component?q=B";
     let request: Request = TestRequest::get().uri(uri).to_request();
     let response: Value = app.call_and_read_body_json(request).await;
-
-    if response["items"][0]["purl"]
-        .as_array()
-        .unwrap()
-        .contains(&Value::from("pkg:rpm/redhat/BB@0.0.0"))
-        || response["items"][1]["purl"]
-            .as_array()
-            .unwrap()
-            .contains(&Value::from("pkg:rpm/redhat/BB@0.0.0"))
-    {
-        assert_eq!(&response["total"], 2);
-    } else {
-        panic!("one of the items component should have matched.");
-    }
-    log::info!("{:?}", response);
+    tracing::debug!(test = "", "{response:#?}");
```

Comment:
nice is using tracing:: this the 'new way' ?

Reply:
It's _a_ way. It's the first way I found that enabled me to easily debug tests without seeing all the instrumentation logs. More details in #1201